### ISSUE TITLE
Other people accepting event invitation results in email saying "invitation updated"

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/server#11539]
originally reported by @jancborchardt 

### Steps to reproduce
1. Create an event and invite other people
2. Have a person confirm their attendance (or reject, I guess)
3. Check your emails

### Expected behaviour
You get an email saying: »Christoph confirmed the invitation« or »Christoph rejected the invitation«

### Actual behaviour
This:
![screenshot from 2018-09-19 21-58-36](https://user-images.githubusercontent.com/925062/45777920-68ba0c00-bc57-11e8-9c33-af5521d38a32.png)

**Comment by jancborchardt** [comment]
@georgehrke are you still the person to look into it? :)

**Comment by georgehrke** [comment]
Yes :)

**Comment by Yannik** [comment]
@georgehrke Thanks for working on this. 
Currently I'm not receiving any emails when a participant accepts my invitation.
Any idea what could cause this?
I've already enabled all the calendar-related email-notification settings in the user settings. However, they're not even showing up in the activity stream, so this does not seem to be an email-related issue. Sending out invites also works fine. I can additionally confirm that the event is correctly updated according to the user selection.

**Comment by kartoffelheinz** [comment]
Anybody still working on this? 

We use NC heavily for recurring events with > 20 participants and the system generates emails "you have been invited" to all of them for each status change. 

Apart from the obvious bug (wrong subject / headline), this behaviour (who receives these attendee-status-changes) should be configurable.

**Comment by georgehrke** [comment]
@kartoffelheinz A pull-request to fix this problem would be much appreciated!

**Comment by Yannik** [comment]
Any progress on this issue? Right now, the invitation email notifications are barely usable..

**Comment by SebastianKrupinski** [comment]
Hi Everyone,

This is proper behavior, according to the iMip RFC (the tech specification how these messages should be sent)  for every status change (user edit or automated like a participant accept/decline) a update message should be sent to all participants with the new information.

We now support automated calendar updates when iMip update messages arrive, so we could auto delete the messages after they are processed.

Let me see what the team thinks? @ChristophWurst @kesselb @st3iny

**Comment by ChristophWurst** [comment]
I don't think automatically deleting user's emails is a good idea.

**Comment by st3iny** [comment]
I agree. Do other clients also delete messages after processing them?

I would argue that some people may want to keep the email as a notification about a changed event.

**Comment by SebastianKrupinski** [comment]
> I agree. Do other clients also delete messages after processing them?

Yes, they do, Outlook and emClient do.

Also, by deleting the messages, I meant from the Inbox (move to trash), not form the server completely.
